src/sidebar/store/modules/activity.js

```diff
@@ -39,6 +43,33 @@ const update = {
     };
   },
 
+  ANNOTATION_SAVE_STARTED(state, action) {
+    let addToStarted = null;
+    if (
+      action.annotation.$tag &&
+      !state.activeAnnotationSaveRequests.includes(action.annotation.$tag)
+    ) {
+      addToStarted = action.annotation.$tag;
+    }
+    const updatedSaves = state.activeAnnotationSaveRequests.concat(
+      addToStarted
+    );
+    return {
+      ...state,
```

Comment:
Do we need the destructing here since our update wrapper does this right?  I just want to make sure we use the right standard on-going.

Reply:
I was following the lead of the other reducers in this module; we should bring this up again at our next frontend sync.

Reply:
Will do, it won't hurt to do this, but it might be good houskeeping to clean this up in a PR across all modules